app/feature/feature-chat/src/main/kotlin/com/hedvig/android/feature/chat/ui/ChatLoadedScreen.kt

```diff
@@ -267,37 +267,41 @@ private fun ChatLazyColumn(
           .padding(bottom = 8.dp),
       )
     }
-    item(
-      key = "Space",
-      contentType = "Space",
-    ) {
-      Spacer(modifier = Modifier.height(8.dp))
-    }
-    if (
-      uiState.fetchMoreMessagesUiState is ChatUiState.Loaded.FetchMoreMessagesUiState.FailedToFetch ||
-      uiState.fetchMoreMessagesUiState is ChatUiState.Loaded.FetchMoreMessagesUiState.FetchingMore
-    ) {
+    // We want to show other items only when there are already some chat messages, to have first messages appear at the
+    // bottom of the UI when the screen is first opened
+    if (uiState.messages.isNotEmpty()) {
```

Comment:
Not placing anything else in the list before the chat messages first show up is the core fix here.
This is the only change here, the rest is just indentation, despite GitHub rendering the diff in a weird way.